machine/build.json

```diff
@@ -1,16 +0,0 @@
-{
-    "image": "homeassistant/{machine}-homeassistant",
-    "shadow_repository": "ghcr.io/home-assistant",
-    "build_from": {
-        "aarch64": "ghcr.io/home-assistant/aarch64-homeassistant:",
-        "armv7": "ghcr.io/home-assistant/armv7-homeassistant:",
-        "armhf": "ghcr.io/home-assistant/armhf-homeassistant:",
-        "amd64": "ghcr.io/home-assistant/amd64-homeassistant:",
-        "i386": "ghcr.io/home-assistant/i386-homeassistant:"
-    },
-    "labels": {
-        "io.hass.type": "core",
-        "org.opencontainers.image.source": "https://github.com/home-assistant/core"
-    },
-    "version_tag": true
```

Comment:
This seems to be dropped, was that intentional?

Reply:
Right, some things from the old days which we never used and solve later with the meta image